mdpow/fep.py

```diff
@@ -579,6 +579,7 @@ def setup(self, **kwargs):
         kwargs['mdrun_opts'] = " ".join([kwargs.pop('mdrun_opts',''), '-dhdl'])
         kwargs['includes'] = asiterable(kwargs.pop('includes',[])) + self.includes
         kwargs['deffnm'] = self.deffnm
+        kwargs['maxwarn'] = kwargs.pop('maxwarn', 1)
```

Comment:
Why pop? Use 
```suggestion
        kwargs.setdefault(‘maxwarn’, 1)
```
... that should be much cleaner. Check the docs for `setdefault`...